### ISSUE TITLE
Defaults

### ISSUE THREAD

**Issue by LiamKearn** [dmarcotte/easy-move-resize#85]
Sorry for question I could probably answer with a bit of a better look.

When I run `defaults export org.dmarcotte.Easy-Move-Resize -` I don't get as many kvs as I expected.

I can see in the codebase the app uses the `userDefaults` interface. I have never touched objective C but my assumption is that I should be able to see all of those properties with defaults?

Thanks for the awesome application btw. It's awesome to see people bringing good old nix customisation to darwin :)

**Comment by LiamKearn** [comment]
Yeah seems to mention it [here](https://github.com/dmarcotte/easy-move-resize/blob/c3fe1dc68883ac3c75cc79246d0052a55f2e4714/easy-move-resize/EMRPreferences.h#L3).

This functionality doesn't seem to be working for me.

**Comment by dmarcotte** [comment]
Hey @LiamKearn, had a look at this, and it seems we're writing to a defaults key called `userPrefs` by accident rather than the intended (and better scoped) `org.dmarcotte.Easy-Move-Resize`.  Try out `defaults read userPrefs` on the command line to see.

Looks like it's been like this for 5 years, so I wonder if it's worth updating...

**Comment by LiamKearn** [comment]
Woo that is quite generic. Don't have my head in the apple ecosystem but I can imagine migrating something as universal as prefs & bundle ids could cause some issues.

I'm happy for now since my `userPrefs` is just this awesome app, Thanks!
